go.mod

```diff
@@ -13,9 +13,9 @@ require (
 	github.com/hashicorp/hc-install v0.6.4
 	github.com/hashicorp/hcl/v2 v2.20.1
 	github.com/hashicorp/logutils v1.0.0
-	github.com/hashicorp/terraform-exec v0.20.0
-	github.com/hashicorp/terraform-json v0.21.0
-	github.com/hashicorp/terraform-plugin-go v0.22.2
+	github.com/hashicorp/terraform-exec v0.20.1-0.20240501214311-a1abb29f777f
+	github.com/hashicorp/terraform-json v0.21.1-0.20240502194054-3b8a921d9133
+	github.com/hashicorp/terraform-plugin-go v0.22.3-0.20240503133000-f7070c19a38c
```

Comment:
TODO: update when `terraform-plugin-go`, `terraform-json` and `terraform-exec` are released

Reply:
This has been updated with the stable releases 🎉